system/include/platform.h

```diff
@@ -357,11 +357,7 @@ typedef int socklen_t;
 
 #define TRUE    1
 #define FALSE   0
-//#define false FALSE
-//#define true  TRUE
 #define HFILE   int
-//#define bool  unsigned char
-#define BOOL    bool
```

Comment:
I assume "BOOL" guaranteed to defined by a standard include file?

Reply:
I think it is only used in windows specific code in the platform.  It is from the days before bool was part of the c/c++ standard...
However I will re-target to master since no need for this change to go in 9.4.x